Gaijinizer patch/CommonEvents_145.txt

```diff
@@ -2,5 +2,5 @@
 > BEGIN STRING
 "Ｐツイスト"
 > CONTEXT: CommonEvents_145/288/InlineScript/1:24
-"P twist"#MTLed
+"The P twist"#Ron
```

Comment:
Should capitalize "Twist" here to match the capitalization on all the other ones.